crates/router/src/connector/utils.rs

```diff
@@ -399,7 +399,7 @@ impl WalletData for api::WalletData {
     fn get_wallet_token(&self) -> Result<String, Error> {
         match self {
             Self::GooglePay(data) => Ok(data.tokenization_data.token.clone()),
-            Self::ApplePay(data) => Ok(data.payment_data.clone()),
+            Self::ApplePay(data) => Ok(data.get_applepay_decoded_payment_data()?),
```

Comment:
will this affect existing users?

Reply:
No ,`get_wallet_token_as_json` function for apple pay is only used in checkout and it has same use case as nexi nets .This is checked with @SamraatBansal  as well .

Reply:
Okay Cool.